chart/kubeapps/values.yaml

```diff
@@ -136,6 +136,26 @@ ingress:
   ##
   secrets: []
 
+## @section Kubeapps packaging options
+## Note: the helm and flux plugins are mutually exclusive, you can only
+## enable one or the other since they both operate on Helm release objects.
+## Enabling carvel or flux does *not* install the required related Carvel or
+## Flux controllers on your cluster. Please read the documentation for running
+## Kubeapps with Carvel or Flux support.
+packaging:
+  ## Default helm plugin
+  ## @param plugins.helm.enabled Enable the standard helm plugin.
+  helm:
+    enabled: true
+  ## Carvel plugin
+  ## @param plugins.carvel.enabled Enable the carvel plugin.
+  carvel:
+    enabled: false
+  ## Flux (v2) plugin
+  ## @param plugins.flux.enabled Enable the Flux plugin.
```

Comment:
```suggestion
  ## @param packaging.flux.enabled Enable the Flux (v2) plugin.
```

Reply:
With the above 3 changes, I additionally updated to avoid the word "plugin" at this level. ie. "Enable support for Flux (v2) packaging."